thirdeye/thirdeye-frontend/app/pods/rootcause/route.js

```diff
@@ -93,54 +51,57 @@ export default Ember.Route.extend(AuthenticatedRouteMixin, {
       metricUrn,
       anomalyUrn,
       session,
-      anomalyContext
+      anomalyContext,
+      anomalySessions
     });
   },
 
   afterModel(model, transition) {
     const maxTime = moment().startOf('hour').valueOf();
 
     const defaultParams = {
-      filters: JSON.stringify({}),
-      granularity: '1_HOURS',
       anomalyRangeStart:  moment(maxTime).subtract(3, 'hours').valueOf(),
       anomalyRangeEnd: moment(maxTime).valueOf(),
       analysisRangeStart: moment(maxTime).endOf('day').subtract(1, 'week').valueOf() + 1,
       analysisRangeEnd: moment(maxTime).endOf('day').valueOf() + 1,
+      granularity: '1_HOURS',
       compareMode: 'WoW'
     };
-    let { queryParams } = transition;
-
-    const validParams = Object.keys(queryParams)
-      .filter((param) => {
-        const value = queryParams[param];
-        return value && isValid(param, value);
-      })
-      .reduce((hash, key) => {
-        hash[key] = queryParams[key];
-        return hash;
-      }, {});
-
-    return Object.assign(
-      model,
-      { queryParams: { ...defaultParams, ...validParams }}
-    );
+
+    // default params
+    const { queryParams } = transition;
+    const newModel = Object.assign(model, { ...defaultParams, ...queryParams });
+
+    // load latest saved session for anomaly
+    const { anomalySessions } = model;
+    if (!_.isEmpty(anomalySessions)) {
+      const mostRecent = _.last(_.sortBy(anomalySessions, 'updated'));
+
+      Object.assign(newModel, {
+        anomalyId: null,
+        anomalyUrn: null,
+        anomalyContext: null,
+        sessionId: mostRecent.id,
+        session: mostRecent
+      });
+
+      // NOTE: apparently this does not abort the ongoing transition
```

Comment:
`transition.abort()` is what you need? 

https://guides.emberjs.com/v2.18.0/routing/preventing-and-retrying-transitions/

Reply:
this seems to be an undocumented quirk. transitionTo() is supposed to abort to ongoing transition, however, it apparently doesn't when only the queryParams change.